### PR TITLE
Update documentation for compiling wasm library

### PR DESCRIPTION
**Pull request by maxammann**
I tried to compile the wasm library myself and failed to compile it with fastcomp.

### CODE REVIEW

**Review by vixentael — APPROVED** [review]

File: themis/languages/wasm/installation.md [hunk 1]
```diff
@@ -78,49 +78,50 @@ WasmThemis uses [BoringSSL](https://boringssl.googlesource.com/boringssl/) as it
 BoringSSL is included as a submodule in the Themis repository,
 but you will need to install its build dependencies as well.
 
- 1. Install Emscripten toolchain
+ 1. Checkout the themis repository.
```
Comment (vixentael) [inline themis/languages/wasm/installation.md:81]:
```suggestion
 1. Checkout Themis repository.
```

File: themis/languages/wasm/installation.md [hunk 2]
```diff
@@ -78,49 +78,50 @@ WasmThemis uses [BoringSSL](https://boringssl.googlesource.com/boringssl/) as it
...
     git submodule update --init
     ```
 
- 4. Make sure that BoringSSL [build dependencies](https://boringssl.googlesource.com/boringssl/+/HEAD/BUILDING.md) are installed.
+ 5. Make sure that BoringSSL [build dependencies](https://boringssl.googlesource.com/boringssl/+/HEAD/BUILDING.md) are installed.
 
     You will typically need to install Go, CMake, GNU Make.
 
- 5. Build and package WasmThemis:
+ 6. Build and package WasmThemis by running the following in the root of the themis repository:
```
Comment (vixentael) [inline themis/languages/wasm/installation.md:124]:
```suggestion
 6. Build and package WasmThemis by running the following in the root of Themis repository:
```

**Comment by vixentael** [comment]
Thank you @maxammann 

Luckily, @ilammy has created https://github.com/cossacklabs/themis/pull/780/ that will allow us to test using 2.0.13 :)

**Comment by ilammy** [comment]
Indeed, support for “fastcomp” has been dropped with https://github.com/cossacklabs/themis/pull/760. Though this change has not been made public in an official release, but builds from `master` branch pretty much require an “upstream” flavor now. Given that the docs describe building the *latest* version from source... Thanks for noticing this, @maxammann!

File: themis/languages/wasm/installation.md (hunk 1 above)
Comment (ilammy) [reply themis/languages/wasm/installation.md:81]: I'm not a native speaker but the “the *Proper-Noun* *of-common-group-noun-kind*” pattern seems to be quite popular. So I'd be happy to just capitalize Themis here.

**Review by ilammy — APPROVED** [review]

File: themis/languages/wasm/installation.md [hunk 3]
```diff
@@ -78,49 +78,50 @@ WasmThemis uses [BoringSSL](https://boringssl.googlesource.com/boringssl/) as it
...
- 2. Activate a suitable toolchain version.
-
-    {{< hint warning >}}
-Currently WasmThemis supports only the **“fastcomp”** flavor of the toolchain.
-The “upstream” (LLVM) flavor has known issues and is not recommended at the moment.
-    {{< /hint >}}
+ 3. Activate a suitable toolchain version.
 
-    Emscripten installs the “upstream” flavor by default
+    Emscripten installs latest “upstream” flavor by default
```
Comment (ilammy) [inline themis/languages/wasm/installation.md:96]: And I guess this should be

```suggestion
    Emscripten installs the latest “upstream” flavor by default
```

Where's @karenswry where we need her? :D

**Comment by maxammann** [comment]
> Thank you @maxammann
> 
> Luckily, @ilammy has created [cossacklabs/themis#780](https://github.com/cossacklabs/themis/pull/780) that will allow us to test using 2.0.13 :)

I saw that PR and already bumped the docs to `2.0.13`. Like a proper timemachine!

**Review by vixentael — APPROVED** [review]

**Pull request merged**